packages/documentation/docs/controls/_number-field_styleguide.mdx

```diff
@@ -0,0 +1,51 @@
+The Number input component allows users to enter and adjust numerical values.
+
+
+![Number input overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3805-24565&t=DtCmoFcLwhf7ke3S-4)
+
+1. Label
+2. Required field indicator (optional)
+3. Value
+4. Stepper buttons (optional)
+5. Input field
+6. Helper or feedback text
+
+## Options
+
+- **Label**: [See form field](./forms/forms-field.md)
+- **Value**: [See form field](./forms/forms-field.md)
+- **Required**: [See form field](./forms/forms-field.md)
+- **Helper text**: [See form field](./forms/forms-field.md)
+- **Feedback text**: [See form field](./forms/forms-field.md)
+- **Show text as tooltip**: [See form field](./forms/forms-field.md)
+- **Placeholder text**: [See form field](./forms/forms-field.md)
+- **Allowed characters pattern**: Define which characters are allowed to be entered. Typically, we use this to reject excluded characters like decimal points with a shaking animation already on value change while typing.
+- **Pattern**: Describe the expected input using regular expressions, for example integer number between 1 and 100. Typically, we use this to validate input on leaving the field or clicking submit.
+- **Min/Max**: Defines the minimum and maximum values allowed
+- **Show stepper buttons**: Optional buttons to increment or decrement the value, suitable for small ranges with few steps
+
+## Behavior in context
+- **Interaction:** Users can type a value or use stepper buttons to adjust it, consider using stepper buttons especially for touch interaction
```

Comment:
Users can type a value or use stepper buttons to adjust it. We typically recommend using stepper buttons, especially for touch interactions, to enhance usability and precision.